config.py

```diff
@@ -0,0 +1,6 @@
+from config import __version__
+
+name='facerec'
+version=__version__
+license='GPL-3'
+url='http://github.com/rushabh-v/linux_face_unlock'
```

Comment:
Remove `from config import __version__` line. And add spaces around operators. e.g. `=`